tests/optimize/test_hyperopt.py

```diff
@@ -413,7 +413,9 @@ def test_log_results_if_loss_improves(hyperopt, capsys) -> None:
         '|   Best |     2/2 |        1 |        0.10% | 0.00100000 BTC |'
```

Comment:
I was thinking it would look more like this.
`0.00100000 BTC (1.00%)`

Reply:
Already changed in latest commit.
```
|        | 2758/5000 |      143 |        0.14% | 0.00118028 BTC ( 19.65%) |         176.5m |     1.99024 |
```

Reply:
Agree. This is still WIP so will make clean up when we are all happy with the outputs :-)